src/s3/util/messages.ts

```diff
@@ -0,0 +1,18 @@
+/*!
```

Comment:
Can this be moved to a more general util file? Nothing here is S3-specific.

Reply:
I think we should move it once we determine whether this a good pattern or not. Most of our errors just say "Error" with some instructing to go to the logs. I'm in favor of standardizing on the pattern here, but I'd like to see it in practice first.